### ISSUE TITLE
Rendered markdown in note list?

### ISSUE THREAD

**Issue by giannissc** [Automattic/simplenote-electron#593]
#### Steps to reproduce
1. Write a markdown note and add (#) to create a heading for the title of the note will show the unrendered markdown on the side note display.

#### What I expected
I would prefer that the side pane also renders the markdown (at least the first few lines that show) to keep consistency with non markdown notes. This is just a suggestion

#### Screenshot / Video
![screenshot from 2017-07-28 00-17-38](https://user-images.githubusercontent.com/20277283/28692726-79077146-732a-11e7-8f21-7f72d897a1f2.png)

**Comment by iblazhko** [comment]
Another option would be to clear Markdown markup so that the notes list would look like this:

    Random Note
    NOTE 1
    ---
    Names
    Planning ahead
    ---
    Book
    Just Listen: ...

**Comment by PlugaruT** [comment]
Any updates in this one?

**Comment by audiovideodisco** [comment]
I would looove it if Simplenote showed the rendered markdown in the note list, as @giannissc suggested. Clearing the Markdown markup, as @iblazhko suggested, would be a fine second choice. 

As it is currently, the markup visible in the note list is so visually janky and hard to parse that I just can't bring myself to use the Markdown in my notes at all. Instead I've searched out a collection of unusual HTML characters that I use for outlining, separating, and emphasizing sections of my notes.

**Comment by rakhi2104** [comment]
I fear a problem, if the title is rendered H1 in few notes in the notes-list and not all, that would look weird.
Instead @giannissc 's hack would be much better.
- One more suggestion would be different titles for each note, so as to differentiate between content of the note and the title itself. :thinking:
